nvtabular/inference/triton/__init__.py

```diff
@@ -70,86 +70,96 @@ def export_tensorflow_ensemble(model, workflow, name, model_path, label_columns,
     tf_config = _generate_tensorflow_config(model, name + "_tf", tf_path)
 
     # generate the triton ensemble
-    ensemble_path = os.path.join(model_path, name)
+    ensemble_path = os.path.join(model_path, name + "_ens")
```

Comment:
I'd rather not change the TF ensemble name if possible (even if it turns out we don't have a choice for hugectr).

Reply:
Cool, just updated it.